Sources/MapboxCoreNavigation/PassiveNavigationEventDetails.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import CoreLocation
+
+struct PassiveNavigationEventDetails: NavigationEventDetails {
+    let coordinate: CLLocationCoordinate2D?
+    let created = Date()
+    let sessionIdentifier = "session-id" // TODO: create global session
+    let driverMode = "freeDrive"
+    
+    var event: String?
+    var userId: String?
+    var feedbackType: String?
+    var description: String?
+    var screenshot: String?
+    // TODO: add time in foreground and background
```

Comment:
Perhaps we can implement this using simply start and end `Date`s, avoiding timers, even if all that gets sent is an elapsed time.

Reply:
I reused the logic we had in `SessionState` class, it doesn't use any timers, just start `Date` and `NotificationCenter` to keep track of iPhone state changes.